ui/plugins/ui/tiled-image-download.plugin.js

```diff
@@ -0,0 +1,326 @@
+;(function(){
+    "use strict";
+    const PAPERSIZE = [
+        {id: "a3p",  width: 297, height: 420, unit: "mm"},
+        {id: "a3l",  width: 420, height: 297, unit: "mm"},
+        {id: "a4p",  width: 210, height: 297, unit: "mm"},
+        {id: "a4l",  width: 297, height: 210, unit: "mm"},
+        {id: "ll", width: 279, height: 216, unit: "mm"},
+        {id: "lp",  width: 216, height: 279, unit: "mm"},
+        {id: "hd", width: 1920, height: 1080, unit: "pixels"},
+        {id: "4k", width: 3840, height: 2160, unit: "pixels"},
+    ]
+
+    // ---- Register plugin
+    PLUGINS['IMAGE_INFO_BUTTONS'].push({ 
```

Comment:
Can this be added next to the other Download buttons, in the horizontal row? That's an array too, so it would be just a push() operation.

Reply:
The label for this button is quite long. It wouldn't fit into that row any more. I can't find a shorter label. Open for suggestions.